alibi_detect/utils/frameworks.py

```diff
@@ -9,3 +9,14 @@
     has_pytorch = True
 except ImportError:
     has_pytorch = False
```

Comment:
This could do with being updated to use `enum`'s, similar to how it is done in alibi:

https://github.com/SeldonIO/alibi/blob/master/alibi/utils/frameworks.py